internal/sgbouncewizard/sgbouncewizard.go

```diff
@@ -242,41 +255,137 @@ func (srv *Server) updateRuleRoute(w http.ResponseWriter, r *http.Request) {
 
 	err = srv.DBClient.UpdateRule(&newRule)
 
+	if err != nil {
+		log.Println(err)
+		http.Error(w, err.Error(), http.StatusNotFound)
```

Comment:
Are we sure this is ***always*** `StatusNotFound`? What if there is a network issue when talking to the database? Is there a way to detect what type of error happened?